src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNode } from "react"
+import { Status } from "./types"
+import styles from "./Toast.module.css"
+
+interface ToastProps {
+  status?: Status
+  message: string
+  action?: ReactNode
+}
+
+export function Toast({ action, message, status = "success" }: ToastProps) {
+  return (
+    <div
+      data-testid={`toast_${message}`}
+      role="alert"
+      aria-live={status === "error" ? "assertive" : "polite"}
+    >
+      <div
```

Comment:
I am likely ignorant here—but could you combine the outer and inner divs' attributes and remove the outer div altogether?